### ISSUE TITLE
bysy list error

### ISSUE THREAD

**Issue by Godje** [Godje/bysy#47]
```
$ bysy list

Traceback (most recent call last):
  File "/home/daniel/.local/bin/bysy", line 11, in <module>
    sys.exit(main())
  File "/home/daniel/.local/lib/python2.7/site-packages/bysy/__main__.py", line 89, in main
    method( args[1].lower() )(args[2:]);
  File "/home/daniel/.local/lib/python2.7/site-packages/bysy/loglist.py", line 9, in loglist
    db = DB().Load();
  File "/home/daniel/.local/lib/python2.7/site-packages/bysy/DB.py", line 85, in Load
    dbloaded = DB.load();
  File "/home/daniel/.local/lib/python2.7/site-packages/bysy/DB.py", line 25, in load
    with open( dbfilename() ) as dbfile:
IOError: [Errno 2] No such file or directory: '/home/daniel/.local/lib/python2.7/site-packages/bysy/db.json'
```

**Comment by Godje** [comment]
The problem is with the `bysy init` not being displayed

**Comment by Godje** [comment]
Fixed in 9ba2af4ddf33436ecf777362b35f301fd15368de
